.github/workflows/commitlint.yml

```diff
@@ -24,4 +25,4 @@ jobs:
         run: npm install --save-dev @commitlint/{config-conventional,cli}
 
       - name: Lint PR title
-        run: echo "${{ github.event.pull_request.title }}" | npx commitlint
+        run: echo "${{ github.event.pull_request.title }}" | npx commitlint@v17.7.2 
```

Comment:
Maybe I'm just unfamiliar with the syntax here, but it looks like we're `npm install`ing commit lint at no-particular-version (so, latest..?) but then we're specifying v17.7.2 when we run it.  Is that right?

What happens when a new version of commitlint gets released?  Wouldn't this `npm install` the newer version but then _also_ try to run commit lint with @v17.7.2?

Reply:
I believe that is correct, good catch. Maybe we can take out that `npm i` completely? I don't think it is doing anything

Reply:
Or, wait, I guess `npm install` is just going to install the version we have stored in the package-lock.json file, innit?  In that case, _why_ do we need to specify the version in our workflow file?

Reply:
this is resolved together through a conversation in gather.